apps/frontend/src/api/util.ts

```diff
@@ -43,3 +47,26 @@ export const useFetchOptions = ({
     },
   };
 };
+
+export interface BasicOptions {
+  searchQuery?: string;
+  filters?: string[];
+}
+
+export const useApiGet = <T>(
+  endpoint: string,
+  parameters: Record<string, string | string[] | undefined> = {},
+  options?: Parameters<typeof useFetchOptions>[0],
```

Comment:
So this is `UseFetchOptions & RequestInit`? :sweat_smile:

Reply:
Yes